akka-distributed-data/src/main/scala/akka/cluster/ddata/LWWRegister.scala

```diff
@@ -48,21 +48,35 @@ object LWWRegister {
   @InternalApi private[akka] def apply[A](node: UniqueAddress, initialValue: A, clock: Clock[A]): LWWRegister[A] =
     new LWWRegister(node, initialValue, clock(0L, initialValue))
 
+  // FIXME when the deprecated functions using Cluster are EOL, add apply's with
+  // implicit: node and node, clock, with defaultClock which the compiler wil not accept until then.
+  def apply[A](node: SelfUniqueAddress, initialValue: A)(implicit clock: Clock[A]): LWWRegister[A] =
```

Comment:
would it be possible to have one with the `defaultClock`. I think it is very rare to replace the `Clock` (only for testing purposes)

Reply:
if problems with overloaded methods it would be ok to have a another factory method for the case when a specific clock is used, `LWWRegister.withClock(node, initialValue, someOtherClock)`

Reply:
It's not possible, after trying to offer it many ways.

Reply:
What does currently work is
```scala
  def apply[A](node: SelfUniqueAddress, initialValue: A): LWWRegister[A] =
    apply(node.uniqueAddress, initialValue, defaultClock)

  // FIXME when the deprecated functions using Cluster are EOL, add apply's with implicit: node, clock
  def apply[A](node: SelfUniqueAddress, initialValue: A)(implicit clock: Clock[A]): LWWRegister[A] =
    apply(node.uniqueAddress, initialValue, clock)
```